### PR TITLE
Finalize Design to Match Zeplin's Design

### PR DESCRIPTION
**Pull request by PabloBona**
###  **Changes Made**

- Added any missing JSX elements to the project to match the design requirements.
- Ensured that all elements required by Zeplin's design are included in the project.
- Styled the website using CSS rules provided in Zeplin to achieve the same styling as the design.
- Verified that the final design of the website matches Zeplin's design, including all elements and styling.
- Update Live Demo

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
+                </Link>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div className="oval">
+          <button type="submit" className="btn btn-sm">
+            <img src={user} alt="" />
+          </button>
         </div>
```
Comment (roshan-bajgain) [inline src/components/Header.jsx]: - [ ] According to the project requirement Styling is **EXACTLY** the same as the one provided on Zeplin. So Please see the design below and try to use the same design. Also, try to use the same color. Everything is provided there color, font etc.
### Your design
his

### Original design
original

File: src/components/BookForm.jsx [hunk 2]
```diff
@@ -80,11 +72,12 @@ function BookForm({ url }) {
...
-          <h2 className="text-secondary my-3 text-uppercase">Add New Book</h2>
-          <form>
+          <hr className="py-2" />
+          <h2 className="text-secondary my-3 text-uppercase fs-4 fw-bolder">Add New Book</h2>
+          <form className="col-form-label-lg">
             <label className="px-3" htmlFor="title">
               <input
-                className="px-2 py-1 rounded border border-secondary"
+                className="px-2 py-1 rounded border border-secondary input-title"
                 type="text"
```
Comment (roshan-bajgain) [inline src/components/BookForm.jsx]: - [ ] According to the project requirement you should use the **EXACTLY** the same as the one provided on Zeplin. See the design below and adjut the spaing and also, the same color. 
- [ ] Button has more padding from left and right please make sure you design match the original design
### Your design
his



### Original design
original

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @PabloBona, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Almost done ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm.jsx (hunk 2 above)
Comment (PabloBona) [reply src/components/BookForm.jsx]: Can you give me the link where it says that the design has to be exactly the same? that's not what my requirements say...
/
Screenshot_20230721-140908
@roshan-bajgain

Comment (roshan-bajgain) [reply src/components/BookForm.jsx]: Hello @PabloBona, It says the `same styling` to the zipline design as it helps you when you work for your client to exactly do the same design as they want. Also, It helps when you go through the job search process. So please try to follow the exact design.

Comment (PabloBona) [reply src/components/BookForm.jsx]: I think that there is a distinction between the phrases 'exactly the same' and 'should match' in the mentioned requirements. I would like to receive guidance on identifying specifically where the word 'exactly' is mentioned in the requirements. Your assistance in this matter would be greatly appreciated.
@roshan-bajgain

Comment (roshan-bajgain) [reply src/components/BookForm.jsx]: please adjust the spacing and also make sure that color matches the design then after that you can ask for another review.

Comment (PabloBona) [reply src/components/BookForm.jsx]: @roshan-bajgain
The power went out all day, just now I was able to make the corrections

File: src/components/BookForm.jsx [hunk 3]
```diff
@@ -80,11 +72,12 @@ function BookForm({ url }) {
...
         <div className="col-12">
-          <h2 className="text-secondary my-3 text-uppercase">Add New Book</h2>
-          <form>
+          <hr className="py-2" />
+          <h2 className="add-tittle">ADD NEW BOOK</h2>
+          <form className="col-form-label-lg add-book">
             <label className="px-3" htmlFor="title">
               <input
-                className="px-2 py-1 rounded border border-secondary"
+                className="px-2 py-2 rounded border border-secondary input-title"
```
Comment (t-yanick) [inline src/components/BookForm.jsx]: - [ ] Great job working on the required changes. There is one more change to make and we would be ready to proceed. Your design is supposed to match the given Zeplin design as indicated by the previous reviewer. See screenshots below:
**Your design**
image
**Zeplin design**
image
When you look closely at the Zeplin design, you will notice that:
- `ADD NEW BOOK` text is aligned with the form element for `Book title`.
- `ADD NEW BOOK` text is aligned with the section above it where there is information about a book.
- The length of the `Category` form is quite long and the `ADD BOOK` button is aligned to the `right` with the section above that has information about a book.
Kindly effect these changes to your design

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @PabloBona , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm.jsx (hunk 3 above)
Comment (PabloBona) [reply src/components/BookForm.jsx]: Hi @t-yanick Is this the wrong alignment in my project? (marked in red in the original?)
image

Comment (PabloBona) [reply src/components/BookForm.jsx]: @t-yanick are  you there ?  Can you plz guide me ?

Comment (t-yanick) [reply src/components/BookForm.jsx]: Hi, I think marked in red is the correct alignment.  The screenshot I added to the review is what I got running your project locally.  If the alignment marked in red is what you have now, kindly request for re-review.  Thanks

**Review by codecaiine — APPROVED** [review]
Hi @PabloBona , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Project well implement ✔️ 
- Routes working fine ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PabloBona** [comment]
@codecaiine Your bookstore-react-redux project is amazing! Thanks for the feedback.

**Pull request merged**